app.rb

```diff
@@ -0,0 +1,45 @@
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  attr_reader :books, :peoples
+
+  def initialize
```

Comment:
- [ ] Kindly add @rentals Otherwise it will not work.
<img width="222" alt="Screenshot 2023-05-27 at 06 33 02" src="https://github.com/imranhossainemi/school_library/assets/81444338/cc7c52ff-8eb8-4570-bdaf-be7152f751a0">